src/redux/actionCreators.js

```diff
@@ -1,4 +1,6 @@
+import React from "react";
 import ACTION_TYPES from "./actionTypes";
+import { usersAPI } from "../api/api";
```

Comment:
Consider fixing the import order and grouping.

Reply:
done